src/main/java/no/nav/pto/veilarbportefolje/kafka/KafkaStats.java

```diff
@@ -4,17 +4,17 @@
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.binder.MeterBinder;
 import io.micrometer.core.lang.NonNull;
+import lombok.RequiredArgsConstructor;
 import no.nav.pto.veilarbportefolje.database.Table;
-import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 
 @Service
+@RequiredArgsConstructor
```

Comment:
Is this required?

Reply:
If it is then @Qualifier("PostgresJdbc") should be added as the table affected by this class is only in postgres